test/local/log_tests.js

```diff
@@ -218,8 +220,14 @@ test(
     }).then(function () {
       t.equal(metricsContext.gather.callCount, 1, 'metricsContext.gather was called once')
 
-      t.equal(logger.info.callCount, 1, 'logger.info was called once')
-      t.equal(logger.info.args[0][1].service, 'bar', 'service property was correct')
+      // activity event data
+      t.equal(logger.info.callCount, 2, 'logger.info was called once')
+      t.equal(logger.info.args[0][1].service, 'baz', 'service property was correct for the activity event')
+
+      // flow event data
+      t.equal(logger.info.args[1][0], 'flowEvent', 'correct op name')
```

Comment:
As a reader of the tests, it feels a bit weird that we don't assert any of the `flowEvent` properties until down here. Any reason not to assert them the first time we encounter them, up in the `'log.activityEvent'` test?